metrics/integration.go

```diff
@@ -61,6 +61,14 @@ var (
 		},
 		[]string{"type", "reason"},
 	)
+
+	ReleaseLatencySeconds = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "release_latency_seconds",
+			Help:    "Latency between integration tests completion and release creation",
+			Buckets: []float64{7, 15, 30, 60, 150, 300, 450, 600, 750, 900, 1050},
```

Comment:
this should be quick operation, I'd use the same buckets as in SnapshotCreatedToPipelineRunStartedSeconds

Reply:
thank you